### DISCUSSION TITLE
How to import .DIA diagrams in Diagrams.net?

### DISCUSSION THREAD

**Discussion by ymike** [jgraph/drawio-desktop#1202] [General]
I have many diagrams created with DIA (0.97.2 for Windows) http://live.gnome.org/Dia
but I am unable to import them in Diagrams.net as a editable diagrams.
I have already Exported in DIA my file in all intermediate not compressed file format (PNG, JPG, WMF, TIFF, VDX, DXF, WPG, SVG, PS, TEX, PDF...) but when I try to import them in Draw.io (20.7.3 for Windows) I always receive the message Error: "Invalid file data" and nothing is imported.
I am able to import the diagrams as a single not-modifiable-image, but not as modifiable file (made up of distinct objects), as I desire.
Any suggestion? Thanks.

**Comment by mod-john** [comment]
I have the same issue, would be interested in importing the Dia file format into draw.io

**Comment by limpus64** [comment]
I've got the same problem here.
